setup.py

```diff
@@ -29,9 +29,9 @@
 from setuptools import find_packages, setup
 
 # NOTE: The following reads the version number and makes
-#       if available to the packaging tools before installation.
-#       REF: https://stackoverflow.com/questions/458550/standard-way-to-embed-version-into-python-package  # noqa: E501
-#       This makes __version__ valid below
+#	if available to the packaging tools before installation.
```

Comment:
I'm not sure what changed here.  Did you add tabs?  If so, please preserve the original spacing format here and elsewhere.